lib/osf-components/addon/components/files/menu/template.hbs

```diff
@@ -3,17 +3,20 @@
     @onClose={{fn this.onCloseMenu @isUploading}}
     as |dropdownMenu|
 >
-    <div local-class='actions-menu-trigger'>
-        <BsButton
-            local-class='trigger-button {{if dropdownMenu.isOpen 'close-button'}}'
-            aria-label={{t 'osf-components.files-widget.expand_files_menu'}}
-            data-ebd-id='{{dropdownMenu.uniqueId}}-trigger'
-            @type='success'
-            {{on 'click' dropdownMenu.toggle}}
-        >
-            <FaIcon @icon='plus' @size='2x' @fixedWidth={{true}} />
-        </BsButton>
-    </div>
+    {{#if this.canEdit}}
```

Comment:
Not sure if disabling the buttons in the menu or hiding the trigger is the way to go. It's a bigger concern though around how we render "unusable" UI.